libs/xml/tinyxml.cpp

```diff
@@ -23,6 +23,8 @@ distribution.
 */
 
 #include <ctype.h>
+#include <clocale>
+#include <iomanip>
```

Comment:
The windows build problem is you need to `#include <sstream>` to get std::stringstream building. Want to add that and push a commit to your branch and we can see the auto-builds again?

Reply:
What is preference, fix this old commit and forced push, or new commit to fix?

Reply:
If you’re comfy with a rebase and force push, that’s better yeah. (Some of our devs aren’t and so I just squash at merge so I’m fine either way)